app/src/main/java/com/duckduckgo/app/browser/BrowserTabViewModel.kt

```diff
@@ -1440,6 +1440,9 @@ class BrowserTabViewModel @Inject constructor(
 
         return if (duckDuckGoUrlDetector.isDuckDuckGoQueryUrl(url)) {
             duckDuckGoUrlDetector.extractQuery(url) ?: url
+        } else if (url.contains("@")) {
```

Comment:
i'm concerned this might affect more URLs than intended; won't this also affect a url like usernames in twitter....
- https://twitter.com/@username
- https://twitter.com/username

in the case of twitter, these are treated as the same, but presume not all sites going to do that, and sometimes the `@` is valid to be included in the URL?

Reply:
Good spot, I forget '@' is allowed in the URI path of URLs. I tested https://medium.com/@duanevalz and the navigations aren't blocked but the address bar is just truncated. I'll use the java URI parser to extract the basic auth component instead.